tests/console/yast2_xinetd.pm

```diff
@@ -27,7 +28,15 @@ sub run {
     script_run("yast2 xinetd; echo yast2-xinetd-status-\$? > /dev/$serialdev", 0);
 
     # check xinetd network configuration got started
-    assert_screen 'yast2_xinetd_startup', 90;
+    assert_screen([qw(yast2_xinetd_startup yast2_xinetd_core-dumped)], 90);
+    if (match_has_tag('yast2_xinetd_startup')) {
```

Comment:
That's wrong. The problem detection should only be called *if* there is a core dump

Reply:
yes, will change in into yast2_xinet_core-dumped.